src/common/Input/Input.jsx

```diff
@@ -1,14 +1,38 @@
-const Input = ({ name, value, labelText, placeholderText, onChange }) => {
+import React from 'react';
+import styles from './Input.module.css';
+
+const Input = ({
+  labelText,
+  placeholderText,
+  inputType,
+  inputValue,
+  inputName,
+  onChange,
+  isRequired,
+  minLength,
+}) => {
+  if (isRequired) isRequired = true;
```

Comment:
`isRequired` is an immutable type (since it props), I cannot understand the purpose of this condition. Let's remove it, please